Engine/src/Engine/Scene.cpp

```diff
@@ -210,9 +209,20 @@ namespace Engine
 				glBindTexture(GL_TEXTURE_2D, texture.texID);
 			}
 
-			if(m_registry.all_of<AnimationComponent>(entity))
+			if (m_registry.all_of<AnimationComponent>(entity))
 			{
-				const auto anim = m_registry.get<const AnimationComponent>(entity);
+				glBindBuffer(GL_ARRAY_BUFFER, vertices.vboID);
+				auto& anim = m_registry.get<AnimationComponent>(entity);
+				anim.deltaTime += dt;
+				
+				double updateTime = 0.3;
```

Comment:
I would expect 0.3 to be a frameRate variable declared in the spriteSheet component

Reply:
similar to spritesOnSheet